src/main/kotlin/dsm/wemeet/global/socket/domain/RoomWebSocketHandler.kt

```
@@ -31,6 +34,16 @@ class RoomWebSocketHandler(
         val roomId = getRoomId(session)
         val peers = roomPeers.computeIfAbsent(roomId) { CopyOnWriteArrayList() }
 
+        try {
+            // 멤버가 이 방에 포함되고 있는지 확인
+            checkIsMemberUseCase.execute(roomId, getUserEmail(session))
+
+            // 이미 세션에 들어와있는지
+            peers.find { getUserEmail(it) == getUserEmail(session) }.let { throw AlreadyJoinedRoomException }
+        } catch (e: Exception) {
```

detekt (1.23.7)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)